contracts/test/integration/IntegrationTestVoltDeployment.t.sol

```diff
@@ -198,9 +201,11 @@ contract IntegrationTestVoltDeployment is DSTest, StdLib {
         uint256 endingPCVDepositFeiBalance = rariFEIPCVDeposit.balance();
 
         assertEq(endingUserVoltBalance - startingUserVoltBalance, mintAmount);
-        assertEq(
-            endingPCVDepositFeiBalance - startingPCVDepositFeiBalance,
-            mintAmount - 1 /// goes down by 1 because of cToken pricing rounding down
+        assertApproxEq(
+            (endingPCVDepositFeiBalance - startingPCVDepositFeiBalance)
+                .toInt256(),
+            (mintAmount - 1).toInt256(), /// goes down by 1 because of cToken pricing rounding down
+            0
```

Comment:
why do we need to assertApproxEq if we just use 0 basis points? it feels like we can just revert this change.

Reply:
noted, fixed